eng/pipelines/release-go-images-pipeline.yml

```diff
@@ -34,6 +44,13 @@ parameters:
     type: string
     default: nil
 
+  - name: notify
+    displayName: >
+      A GitHub username to @mention in the tracking issue to subscribe them to updates. Do not
```

Comment:
this might not be the best description for this pipeline. it's more about being used to add your name as the author of the blog

Reply:
Perhaps change this display name to:

A GitHub username which is mapped to a Wordpress username for authoring the release blog posts.

Reply:
fixed